decoder/pokestop.go

```diff
@@ -583,15 +592,21 @@ func (stop *Pokestop) updatePokestopFromGetContestDataOutProto(contest *pogo.Con
 		focussedPokemon = contest.GetFocus().GetPokemon()
 	}
 
-	if focussedPokemon == nil {
+	if focussedPokemon == nil && focussedPokemonType == nil {
```

Comment:
We should probably separate that check, check for pokemon and type separately.
Only one of it might be active and the other might need to be reset, right?

Reply:
I had that in mind, but it seem to default to null (from what I can understand), if the value is not set.
So that's why I didn't add it as a separate check, but can be added as a seperate check, and reset if it's prefered.

Reply:
focussedPokemon and focussedPokemonType defaults to null, that's true. But we also need to reset the value on stops.

It might not be an issue with the actual way how showcases work, but imagine next Showcase eould be pokemon again, then the pokemon type would be still stored in db :)